tests/test_CSVTimeSeriesReader.py

```diff
@@ -414,6 +414,22 @@ def test_country_lookup(self):
             data = ts.data(vars[0])
         self.assertTrue(False)
 
+    def test_elevation_filter_1(self):
+        engines = pyaro.list_timeseries_engines()
+        with engines["csv_timeseries"].open(
+            filename=self.file,
+            filters=[pyaro.timeseries.filters.get("altitude", max_altitude=-5)],
+        ) as ts:
+            self.assertEqual(len(ts.stations()), 0)
+
+    def test_elevation_filter_2(self):
+        engines = pyaro.list_timeseries_engines()
+        with engines["csv_timeseries"].open(
+            filename=self.file,
+            filters=[pyaro.timeseries.filters.get("altitude", min_altitude=5)],
+        ) as ts:
+            self.assertEqual(len(ts.stations()), 0)
```

Comment:
Both tests remove all stations. This is a very weak test. You may update the input data so that you can generate better outcomes, e.g. filter only some of the stations and keep some others?